tooling/lib/openh264.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const debug = require(`debug`)(`tooling:openh264`);
+const denodeify = require(`denodeify`);
+const {rimraf} = require(`./async`);
+const spawn = require(`../util/spawn`);
+const FirefoxProfile = require(`firefox-profile`);
+const os = require(`os`);
+const path = require(`path`);
+const {stat} = require(`fs-promise`);
+
+const PROFILE_DIR = `./.tmp/selenium`;
+
+const copy = denodeify(FirefoxProfile.copy);
+/**
+ * denodeifies FirefoxProfile.encode
+ * @param {FirefoxProfile} fp
+ * @returns {Promise<string>}
+ */
+function encode(fp) {
+  return new Promise((resolve, reject) => {
+    fp.encode((err, encoded) => {
+      if (err) {
+        reject(err);
+        return;
+      }
+      resolve(encoded);
+    });
+  });
+}
+
+/**
+ * rsyncs a directory
+ * @param {string} src
+ * @param {string} dest
+ * @returns {Promise}
+ */
+async function rsync(src, dest) {
```

Comment:
Please open a backlog item to audit calls to `spawn` for cross-platform compatibility.